.github/workflows/release.yml

```diff
@@ -39,5 +39,12 @@ jobs:
         if: steps.changesets.outputs.published == 'true' || github.event_name == 'workflow_dispatch' # run when the package is published via changeset or if a release is triggered manually.
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        run: yarn publish-npm
+      - name: Build Blade if not built by changeset
+        if: steps.changesets.outputs.published == 'false'
```

Comment:
When will this happen, wouldn't it always be built in release 🤔

Reply:
Release workflow always runs but it only builds when blade is to be published that is when update version PR is merged otherwise it doesn't build.

PS: I've added this as my best understanding of the workflow, whether it will work as intended or not we'll know only when we merge it to master 🤡 

But it should work. I'll monitor it

Reply:
yolo

Reply:
Why do we need to build blade in release workflow?